mp4/file.go

```diff
@@ -105,47 +116,34 @@ func (f *File) AddMediaSegment(m *MediaSegment) {
 }
 
 // DecodeFile - parse and decode a file from reader r
-func DecodeFile(r io.Reader) (*File, error) {
+func DecodeFile(r io.Reader, options ...Option) (*File, error) {
 	f := NewFile()
+
+	// apply options
+	f.ApplyOptions(options...)
```

Comment:
Either explain more, or remove the comment...

Reply:
updated accordingly